### ISSUE TITLE
s3 links don't work. 

### ISSUE THREAD

**Issue by fessyfoo** [xironix/freebsd-vagrant#16]
s3 links don't work. 
google cloud storage links not up?

is this project dead?

**Comment by nathany** [comment]
:question:

**Comment by krzysztofantczak** [comment]
Yup, there is no way to download this box. Are there any mirrors of it?

**Comment by nathany** [comment]
I'm not sure if there are any mirrors, but a friend pointed me to Pete Cheslock's BSD boxes:

https://gist.github.com/petecheslock/d7394ff93ce783c311c7
